verilog/CST/expression.cc

```diff
@@ -72,6 +72,32 @@ bool ConstantIntegerValue(const verible::Symbol& expr, int* value) {
   return absl::SimpleAtoi(text, value);
 }
 
+
+const verible::Symbol* UnwrapExpression(const verible::Symbol& expr) {
+  
+  const auto& node = verible::SymbolCastToNode(expr);
```

Comment:
This will terminate with an assert failure if passed a leaf, so suggest checking `expr.Kind()` first before doing this cast.